src/vs/workbench/common/theme.ts

```diff
@@ -182,6 +182,17 @@ export const TAB_UNFOCUSED_HOVER_BORDER = registerColor('tab.unfocusedHoverBorde
 
 //#endregion
 
+//#region Tab Drag and Drop Indicator
+
+export const TAB_DRAG_AND_DROP_BETWEEEN_INDICATOR = registerColor('tab.dragAndDropBetweenIndicator', {
```

Comment:
A few things:
- can we see how the color is declared for when you drag views or panels so that we align the two, possibly even share the same base color?
- I am not sure about "between indicator", to me this color is a border, I would see to pick a name that is similar to existing theme names in that regard
- do we no longer use colors for drag and drop feedback that we should remove?

Reply:
The other dnd borders use the foreground color as the base color which is why I decided to do the same. All colors are still used except some high contrast theme colors which I am still looking into